### ISSUE TITLE
Disable .hidden position after the initial launch

### ISSUE THREAD

**Issue by Isuru-Nanayakkara** [SCENEE/FloatingPanel#344]
### Description

Initially the floating panel is hidden. When a button is tapped, it comes into the half position. Then user can go back and forth between full and half by swiping. But since initially hidden was also added as a supported position, if I swipe down while the panel is in the half position, it slides down and goes into the hidden position.

### Expected behavior

I want to have the panel hidden initially but after showing it, I want to keep it between half and full positions.
### Steps to reproduce

```swift
class CumtomFloatingPanelLayout: FloatingPanelLayout {
    var initialPosition: FloatingPanelPosition {
        return .hidden
    }
    
    var supportedPositions: Set<FloatingPanelPosition> {
        return [.hidden, .half, .full]
    }
    
    func insetFor(position: FloatingPanelPosition) -> CGFloat? {
        switch position {
            case .full: return 200
            case .half: return 240
            case .hidden, .tip: return nil
        }
    }
}
```
**How do you display panel(s)?**

- [ ] Add as child view controllers
- [x] Present modally

**How many panels do you displays?**

- [x] 1 
- [ ] 2+

### Environment

**Library version**
1.7.4

**Installation method**

- [x] CocoaPods
- [ ] Carthage
- [ ] Swift Package Manager
 
**iOS version(s)**
13.4

**Xcode version**
11.4.1

**Comment by SCENEE** [comment]
I understood your use case. I will consider how to deal with it in the library.

**Comment by SCENEE** [comment]
@Isuru-Nanayakkara I created #345 PR to support your use case. Could you try it?

**Comment by Isuru-Nanayakkara** [comment]
Thanks for the quick fix. I tried it and it seems to work great! 👍🏼

**Comment by SCENEE** [comment]
Thanks for your confirmation! It will be released in v1.7.5.
